app/assets/javascripts/application.js.erb

```diff
@@ -79,7 +79,7 @@ $(document).on('turbolinks:load', function() {
 
   // Modal windows
   $('.open_modal').magnificPopup({
-    type: 'inline',
+    type: 'ajax',
```

Comment:
Careful, changing the behaviour here breaks other parts of the code which rely on this magnificPopup, for example, editing a stage of a process:

![screen shot 2017-09-14 at 15 52 50](https://user-images.githubusercontent.com/17616/30433454-cb7aa49c-9964-11e7-8410-b7cc8480cde0.png)

I'd suggest to have to handlers:

- `$('.open_modal')` opens an inline modal
- `$('.open_remote_modal)` opens a remote modal, using AJAX

Reply:
Sorry 😓  , I didn't see that part. I saw that it was used in this file (app/Views/gobierto_budgets/budgets/index_old.html.erb)  but I thought it was deprecated. I added open_remote_modal to the application javascript because it was only for admin application javascript.
`
link_to new_user_sessions_path(open_modal: true), class: "pure-menu-link open_remote_modal"
`
Thanks!!